src/products/l10n_epms.md

```diff
@@ -0,0 +1,29 @@
+# Localization team ePMs
+
+## Bryan
+
+* [Email](bolsson@mozilla.com)
```

Comment:
```suggestion
* Email: bolsson(at)mozilla.com
```

These aren't links, you can preview the page
https://github.com/Delphine/localizer-documentation/blob/add_pm/src/products/l10n_epms.md